### PR TITLE
Initial VollaOS for vidofnir

### PR DESCRIPTION
**Pull request by luka177**

### CODE REVIEW

File: v2/devices/vidofnir.yml [hunk 1]
```diff
@@ -135,3 +135,73 @@ operating_systems:
...
+                  file: "unpacked/vendor_boot.img"
+                  group: "firmware"
+      - actions:
+          - fastboot:erase:
+              partition: "userdata"
+      - actions:
+          - fastboot:reboot:
+        fallback:
+          - core:user_action:
+              action: "boot"
```
[the remark this answers is not in the document]
Comment (luka177) [reply v2/devices/vidofnir.yml:202]: Was that some recent change?

File: v2/devices/vidofnir.yml [hunk 2]
```diff
@@ -135,3 +135,73 @@ operating_systems:
...
+    slideshow: []
+    eula:
+      title: "EULA"
+      description: "THE TERMS OF USE OF THE VOLLA OS ONLY ALLOW AN INSTALLATION ON A CLEARLY BRANDED VOLLA PHONE X 23. To proceed with the installation you have to confirm that you have read and understood the End User License Agreement (EULA) of Hallo Welt Systeme UG (haftungsbeschrÃ¤nkt) for the Volla OS and agree to it."
```
[the remark this answers is not in the document]
Comment (luka177) [reply v2/devices/vidofnir.yml:206]: Ouch, its incorrect anyone ill take care of it

File: v2/devices/vidofnir.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (amartinz) [reply v2/devices/vidofnir.yml:202]: It never existed, when [looking at the history](https://github.com/ubports/installer-configs/commits/master/v2/devices/vidofnir.yml).
This is not a common action but defined per device/config.

File: v2/devices/vidofnir.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (luka177) [reply v2/devices/vidofnir.yml]: Fixed

File: v2/devices/vidofnir.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (luka177) [reply v2/devices/vidofnir.yml:202]: ah its per device, ok sorry for confusion

**Comment by amartinz** [comment]
Thanks!

@JamiKettunen could you please have a look as well? :)

Given [this PR](https://github.com/ubports/installer-configs/pull/238) maybe it makes also sense to add the slot suffix to partitions?

**Comment by J0J0** [comment]
FYI: Worked for me for going UT->VOS (version of 7c1bc4c4baf71c56c40a00455729f2a3e2747019). :)

**Comment by JamiKettunen** [comment]
@amartinz UT already has `_a` and the VP22 PR doesn't touch VollaOS either so this should be fine as is (fwiw I also tested this)

**Pull request merged**